foundation/foundation-upgrade/src/test/java/com/boozallen/aissemble/upgrade/migration/v1_7_0/DataLineageMigrationSteps.java

```diff
@@ -0,0 +1,62 @@
+package com.boozallen.aissemble.upgrade.migration.v1_7_0;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Upgrade
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import com.boozallen.aissemble.upgrade.migration.AbstractAissembleMigration;
+import com.boozallen.aissemble.upgrade.migration.AbstractMigrationTest;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+import java.io.File;
+import java.nio.file.Files;
+
+import static junit.framework.TestCase.assertTrue;
+
+public class DataLineageMigrationSteps extends AbstractMigrationTest {
+    private AbstractAissembleMigration migration;
+    File validatedFile = getTestFile("/v1_7_0/DataLineageMigration/validation/CdiContainerFactory_Validation.java");
```

Comment:
A:
```suggestion
    private File validatedFile = getTestFile("/v1_7_0/DataLineageMigration/validation/CdiContainerFactory_Validation.java");
```
but why not make this a local variable?